rancher-vcluster/rancher-vcluster.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: harvesterhci.io/v1beta1
+kind: Addon
+metadata:
+  name: rancher-vcluster
+  namespace: harvester-system
```

Comment:
should we consider putting this in an independent namespace like `rancher-vcluster-system`

Reply:
+1 to this if that works. @guangbochen do we need UI's help to add a new system namespace?

Reply:
or it can be added as a YAML file directly in here, unless we like users to define the namespace specifically.

Reply:
i will just add it to the yaml